### PR TITLE
Revert "Revert "change teacher announcement from purple to blue, add more prominent l…""

### PR DESCRIPTION
**Pull request by tanyaparker**
Reverts code-dot-org/code-dot-org#9906

The problem: The teacher announcement section was originally only on the teacher dashboard page. Since we added it to also be on the Code Studio homepage, https://github.com/code-dot-org/code-dot-org/pull/9881 refactored the teacher announcement css to be shared. Including teacher dashboard css on the dashboard side messed up dashboard headers. See the minecraft headers below with `overflow: hidden` added to their styles.

The fix: Separate out teacher announcement css into its own scss file (like the user hero that appears on Code.org homepage and below the teacher announcement on Code Studio landing page). Now only the necessary css is shared.

**Before**
image

**After**
image

**Teacher announcement on Code Studio**
image

**Teacher announcement on teacher dashboard**
image

### CODE REVIEW

**Comment by breville** [comment]
lgtm.  sorry for not catching this the first time.

File: shared/css/teacher-announcement.scss [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    .content-text {
+      width: 900px;
+
+      a {
+        text-decoration: none;
+        color: $cyan;
+      }
+
+      .fa {
+        margin: 0px 10px 0px 5px;
```
Comment (ashercodeorg) [inline shared/css/teacher-announcement.scss]: nit: s/`0px`/`0`.

**Comment by ashercodeorg** [comment]
LGTM.

**Comment by ashercodeorg** [comment]
This seems not to have changed any eyes tests (per early AM DTT run on 2016-08-09). Is this a concern?

**Comment by tanyaparker** [comment]
If I'm reading this right, I think the only eyes test for logged in Code Studio homepage is only tested as a student? So they won't see any announcement changes. I also don't see a test for teacher dashboard. Should I add tests for both scenarios? https://github.com/code-dot-org/code-dot-org/blob/363bcf2402f0bfe51ab70c6c9cc709ac46cec27b/dashboard/test/ui/features/eyes.feature#L224

**Pull request merged**